pages/index.js

```diff
@@ -1,14 +1,16 @@
 import ArticleList from "../components/ArticleList";
-import styled from "styled-components";
 import useSWR from "swr";
 import { fetcher } from "../helpers/api";
 
 function Home() {
   const { data: articles, error } = useSWR("/api/articles", fetcher);
 
   if (!articles) {
-    return <h4>loading...</h4>;
+    return <h4>Artikel Details werden geladen...</h4>;
```

Comment:
Why do you use a `h4` tag here? Why not start with `h1` ?

Reply:
h1 is in the headline ("Bazam", "Details"). I thought there must be only one `h1` tag for eache page